### ISSUE TITLE
CC-BY-ND icon issue: not free/libre

### ISSUE THREAD

**Issue by hosiet** [robert7/nixnote2#66]
Looks like icons in resources/icons/* are noted to be CC-BY-ND 3.0... Oops.

**"ND"** is a giant issue. That essentially made it incompatible with Debian since it disallows modification, which contradicts with Debian's DFSG guideline.

I guess this issue has to be solved sooner or later. I'm wondering your attitude towards it and we may be able to come up with an acceptable way to solve it.

BTW, here's an (not-so-complete) list of DFSG-compatible and -incompatible licenses: https://wiki.debian.org/DFSGLicenses

**Comment by robert7** [comment]
ok. I did not know..

I used just one icon from icon8.

the icon is free to use https://icons8.com/license, the licence is "CC-BY-ND"

but they also say, icons for open source are free - https://icons8.com/articles/weve-contributed-an-icon-to-imageoptim/ - the project should be compliant - I can try to write them

if they decline, I can buy 1 month subscription, then I can use the icon without notice - https://icons8.com/paid-license-99/#/

**Comment by hosiet** [comment]
Sure. Please let me know once there's any progress.

The target is that the icon we use must be released under a free/libre license without restriction of who is using it or limits on its redistribution/modification.

The detailed approach to reach that goal can vary (e.g., buy it entirely and re-license it by yourself or reach agreement with its author to re-license it under a DFSG-compatible license, etc.)

**Comment by robert7** [comment]
ok. i've just sent them email. 

in worst case I can also replace the icon, I have no big attachment to it.

**Comment by hosiet** [comment]
Note: even if you buy a 1-month subscription, the agreement still declares that you cannot re-sell and cannot use it under certain circumstances thus buying subscription cannot solve the problem.

**Comment by robert7** [comment]
OK. let wait what they reply.

**Comment by robert7** [comment]
@hosiet how much urgent is the problem?
I asked a fried to draw a new icon... but this will take some time as he is quite busy

but I could quickly produce something myself, similar to the existing icon (I already found some other icon which is licenced "CC 3.0 BY" which I can remix into something similar to existing one). As I understanded "CC 3.0 BY" would be ok, right?

so one way or another, the problem will be resolved in next few days

**Comment by hosiet** [comment]
@robert7 CC-BY is okay as long as it is v3.0, v4.0 or higher. CC-BY v1.0 or v2.0 is not OK. See https://wiki.debian.org/DFSGLicenses#Creative_Commons_Attribution_License_.28CC-by.29.2C_v1.0 for details.

I'm not really in a hurry but it's much better if we fix it before 2019 comes.

**Comment by robert7** [comment]
OK. I already got reply from the icons8.com where they say, they are OK with me using the icon for the project. I just need one more clarification from their side..

Additionally I already created another replacement icon as backup, which is very similar as the current one, where the source has "CC 3.0 BY licence.

Plus next week I should get new icon suggestion from the designer friend.
So this will be fixed in next days.
